### ISSUE TITLE
Endowments cancelling things should notify the user

### ISSUE THREAD

**Issue by ritave** [MetaMask/snaps-monorepo#953]
There should be a console warning when things are cancelled due to endowments teardown.

Context #952

**Comment by Montoya** [comment]
Is this a v1 requirement?

**Comment by ritave** [comment]
No, we can add this later when we're focusing on experience of the development.
It should be a minor change in any cas
